src/menu/src/MenuItem.js

```diff
@@ -116,6 +99,8 @@ MenuItem.propTypes = {
    */
   is: Box.propTypes.is,
 
+  className: PropTypes.string,
```

Comment:
Can you leave the same comment for this prop discouraging use of `className`? We use this throughout the codebase, and is something that I'd love to keep pushing for.

Reply:
Can add the description!